thread.c

```diff
@@ -1156,7 +1156,7 @@ int _mutt_traverse_thread(struct Context *ctx, struct Header *cur, int flag)
   struct MuttThread *thread = NULL, *top = NULL;
   struct Header *roothdr = NULL;
   int final, reverse = (Sort & SORT_REVERSE), minmsgno;
-  int num_hidden = 0, new = 0, old = 0;
+  int num_hidden = 0, new = 0, old = 0, flagged = 0;
   int min_unread_msgno = INT_MAX, min_unread = cur->virtual;
```

Comment:
While you're editing this file, please could you change `new`, `old` and `flagged` into `bool`, then replace `0`/`1` with `false`/`true`.